deploy/README.md

```diff
@@ -0,0 +1,3 @@
+# Tools for deployment and distributions
```

Comment:
For now, can we move the `linux-systemd` folder to `tools/systemd` and get rid of the top level `deploy/` directory?